discord-service/src/main/resources/application.yml

```diff
@@ -2,6 +2,14 @@ discord:
   token: DISCORD_BOT_TOKEN
   enabled: false
 
+  links:
```

Comment:
Wiem, że to jest interpretowane jako Map<String, String> ale mam wrażenie że lepiej by wyglądało tak
```yaml
links:
  - alias: Strona internetowa
    url: https://flyingoctopus.pl
  - alias: Facebook
    url: https://www.facebook.com/FO.flyingoctopus/
```